### ISSUE TITLE
Applications start on top of each other randomly

### ISSUE THREAD

**Issue by ghost** [hyprwm/Hyprland#6185]
### Hyprland Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch  at commit cba1ade848feac44b2eda677503900639581c3f4  (props: bump version to 0.40.0).
Date: Sat May 4 15:42:32 2024
Tag: v0.40.0, commits: 4606

flags: (if any)
Arch Linux x86_64
```

</details>

### Bug or Regression?

Bug

### Description

I started using Hyprland after months, sometimes, windows randomly launch stacked on top of each other andd the only way to fix it is to close the window.

### How to reproduce

I tried updateing Hyprland as someone suggested from the server, but the issue still persists.

### Crash reports, logs, images, videos

https://github.com/hyprwm/Hyprland/assets/78359804/0408c722-7bdb-4bbc-bf2d-68bec229fbf5

**Comment by vaxerski** [comment]
is this reproducible? if not, we can classify this as ghosts in your computer

**Comment by vaxerski** [comment]
every time? can I get your config and exact repro steps?

**Comment by Schwarzeisc00l** [comment]
No, not every time. It just happens randomly. [Here's my config](https://privatebin.net/?c6998f10e3595e98#6QPfe7JAohSK35oEMDfDNBaWuesePUduRu3dH6udDzP7)

**Comment by hjpaul7** [comment]
I get this as well. It sends windows to workspace "-98" when I check hyprctl clients.

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
